src/OpenIddict.Core/OpenIddictBuilder.cs

```diff
@@ -431,7 +432,7 @@ public class OpenIddictBuilder {
         /// </summary>
         /// <returns>The <see cref="OpenIddictBuilder"/>.</returns>
         public virtual OpenIddictBuilder UseJsonWebTokens() {
-            return Configure(options => options.UseJwtTokens());
+            return Configure(options => options.AccessTokenHandler = new JwtSecurityTokenHandler());
```

Comment:
@ilmax FYI, ASOS beta6 no longer have a `UseJwtTokens()` extension to encourage people to use the opaque format (yeah, I'm evil :smile:)